test/TestExpectations.json

```diff
@@ -3,7 +3,7 @@
     "testIdPattern": "*",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["webDriverBiDi"],
-    "expectations": ["SKIP", "TIMEOUT"]
```

Comment:
the expectations are not ready for review, right?

Reply:
Yes, they aren't ready for review.